src/go/rdctl/pkg/config/config.go

```diff
@@ -51,17 +54,24 @@ var (
 	configDir         string
 	configPath        string
 	defaultConfigPath string
+	configFile        = "rd-engine.json"
+	rdSubDir          = "rancher-desktop"
 )
 
 // DefineGlobalFlags sets up the global flags, available for all sub-commands
 func DefineGlobalFlags(rootCmd *cobra.Command) {
 	var err error
-
-	configDir, err = os.UserConfigDir()
-	if err != nil {
-		log.Fatal("Can't get config-dir: ", err)
+	if runtime.GOOS == "linux" && isWSLDistro() {
+		if defaultConfigPath, err = wslifyConfigPath(); err != nil {
+			log.Fatalf("Can't get config-dir: %v", err)
```

Comment:
I would make the error message subtly different, so you can tell that the code took the WSL branch and not the regular one, e.g.

```suggestion
			log.Fatalf("Can't get WSL config-dir: %v", err)
```